modules/test/playwright/tests/frontend-data-set-views-web/pages/view/visualization_modes/VisualizationModesPage.ts

```diff
@@ -108,6 +133,6 @@ export class VisualizationModesPage {
 			})
 			.click();
 
-		await this.toastContainer.isHidden();
+		await expect(this.toastContainer).toBeHidden();
```

Comment:
In this case, `not.toBeInViewport()` did not work. It looks like it's not a simple isVisible bad - inViewport good. It depends on the specific case. The real confirmation is only if playwright shows the expected state on an assertion.